src/elements/content-sidebar/activity-feed/task-form/TaskForm.scss

```diff
@@ -53,6 +53,17 @@
             .pill-selector-input-wrapper {
                 width: auto;
             }
+
+            .overlay-wrapper {
+                border: 1px solid $sf-fog;
+                height: 215px;
```

Comment:
let's pull this up as a constant. if it can be derived or related to other measurements, we can set those as well.